iowrappers/utils.go

```diff
@@ -33,9 +33,9 @@ func scanRedisKeys(context context.Context, redisClient *RedisClient, redisKeyPr
 	return redisKeys, nil
 }
 
-// a generic filtering function for places
-func filter(places []POI.Place, condition func(place POI.Place) bool) []POI.Place {
-	var results []POI.Place
+// Filtering function for places using generics
```

Comment:
Nit: comment should start with the function name in Go.

Reply:
Done